core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -184,6 +188,18 @@ var ContributorDashboardAdminPage = function() {
     await _assignContributionRights(username, CATEGORY_SUBMIT_QUESTION);
   };
 
+  this.switchLanguage = async function(language) {
+    await action.click('Language Selector', languageSelector);
+    await waitFor.visibilityOf(
+      languageDropdown, 'Language Dropdown not visible');
+    var selectorOption = languageSelector.$(
+      `.e2e-test-language-selector-option=${language}`);
+    await action.click(`${language} option selector`, selectorOption);
+    await action.click('Language Selector', languageSelector);
+    await waitFor.invisibilityOf(
+      languageDropdown, 'Language Dropdown did not disappear');
```

Comment:
USe action.matSelect*

Reply:
We can't use `action.select/action.matselect` here this is the same situation we discussed in the other PR.[ Reference](https://github.com/oppia/oppia/pull/18951/files/394ac59329e1651784703689bc8577703cc4ce2d#r1342832961). We can only use action.select when we have implemented <select> in the code. If we are handling dropdown with a div we can't use it.